### DISCUSSION TITLE
Multi Cursors Incremental Expand and Edit

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#708] [Q&A]
Is there a way to select first `apply_bonus` and then expand the selection to the other `apply-bonus` and edit it with meow's multi-cursors? 🦆 

```python
def calculate_total_score(scores):
   base_score = sum(scores)
   return apply_bonus(base_score)

def apply_bonus(score):
   if score > 100:
       return score * 1.1
   return score
```

I kind of want something like `M-s .`, filter those matches, and finally multi edit. 

[vis](https://github.com/martanne/vis) can do this with elegant keybindings.

**Comment by Esnos33** [comment]
If you want to use search/replace, just use `meow-query-replace`. If you want something more than search/replace, than you have to use beacons
